brownie/_cli/test.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/python3
 
+import os
```

Comment:
Would prefer to use [`pathlib.Path`](https://docs.python.org/3/library/pathlib.html#pathlib.Path) over `os.path` to maintain consistency with the rest of the code base.